src/Components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Display extends Component {
+  static defaultProps = { name: 'My Calculator' };
+
+  render() {
+    return <div className="Display"><h1> { this.props.name } </h1></div>;
+  }
+}
```

Comment:
- Lets fix this please, we are not using state in this component so, class-based should be replaced with functional compoenent.
- It should be taking the result of a calculation as an incoming prop. 
![image](https://user-images.githubusercontent.com/46115725/89368805-0418c680-d6a2-11ea-8d04-cb8ebd7c35f0.png)
You should have a result prop instead of a name prop, also the default value of it should be set to O.